pkgs/applications/window-managers/i3/lock-color.nix

```diff
@@ -25,6 +25,8 @@ stdenv.mkDerivation rec {
   installFlags = "PREFIX=\${out} SYSCONFDIR=\${out}/etc MANDIR=\${out}/share/man";
   postInstall = ''
     mv $out/bin/i3lock $out/bin/i3lock-color
+    mv $out/share/man/man1/i3lock.1 $out/share/man/man1/i3lock-color.1
+    sed -i 's/^\.B i3lock$/.B i3lock-color/' $out/share/man/man1/i3lock-color.1
   '';
   meta = with stdenv.lib; {
     description = "A simple screen locker like slock";
```

Comment:
While you're at it, can you please add a hint here on how this differs from regular `i3lock`, for example
"A simple screen locker like slock, enhanced version with extra color options" 
(feel free to come up with a better wording).

Reply:
Ok. This will be in a separate pull-request.